.github/workflows/ci.yml

```diff
@@ -100,11 +100,15 @@ jobs:
         with:
           go-version-file: "go.mod"
       - name: Run e2e tests
-        uses: ava-labs/avalanchego/.github/actions/run-monitored-tmpnet-cmd@cc3242f48a6262c52057e98a85d6cf8184dfaca3
+        uses: ava-labs/avalanchego/.github/actions/run-monitored-tmpnet-cmd@36baa0509d61679085d35bffa590d63732b0a7bd
         with:
           run: ./scripts/run_task.sh test-e2e-ci
+          prometheus_url: ${{ secrets.PROMETHEUS_URL || '' }}
```

Comment:
Are these proetheus/loki changes required because of the AvalancheGo update in this PR? Otherwise seem like they could be made separately